src/JsExpression.php

```diff
@@ -21,13 +21,7 @@ class JsExpression implements JsExpressionable
      */
     public $args = [];
 
-    /**
-     * Constructor.
-     *
-     * @param string $template
-     * @param array  $args
-     */
-    public function __construct($template = '', $args = [])
+    public function __construct(string $template = '', array $args = [])
```

Comment:
array tag type is super important - otherwise string can be passed without any warning!